chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -173,13 +173,17 @@ private User create(final CreateUserFromInvitationRequest userInfo)
             validationErrors.add(errorMessageUtil.getMessage("user.fullName.maxlength",
                     getMaxLength("fullName")));
         }
-        if (request.getUser().getFriendlyName().length() > getMaxLength("friendlyName")) {
-            validationErrors.add(errorMessageUtil.getMessage("user.friendlyName.maxlength",
-                    getMaxLength("friendlyName")));
+        if (!StringUtils.isEmpty(request.getUser().getFriendlyName())) {
```

Comment:
Why not do this as an `&&`, as well as the second one?
```suggestion
        if (!StringUtils.isEmpty(request.getUser().getFriendlyName())
```